systers_portal/templates/meetup/list_meetup.html

```diff
@@ -5,35 +5,46 @@
 {% endblock %}
 
 {% block content %}
-  <style type="text/css">
-  </style>
-  
-  <h1 class="mt40 text-center meetup-list-head">Upcoming Meetups</h1>
-  <p class="meetup-list-subhead">We host meetups, workshops, networking events, and more. See what’s happening near you.</p>
-    
-`<div class="search-options">
-      <div id="meetup-location" class="fake-border">
-        <i class="glyphicon glyphicon-flag"></i>
-        
-        
-      <div id="keyword" class="fake-border">
-        <i class="glyphicon glyphicon-search"></i>
-        <input id="keyword-input" type="text" name="keyword" placeholder="Keyword"/>  
-      </div>
-      <input id="go-btn" class="search-btn" type="submit" name="submit" placeholder="Go"/>
-                <select id="location-input" name="location" placeholder="Location">
-        {% for city in cities_list %}
-        <option value = "{{forloop.counter}}">{{ city }}</option>
-
-        {% endfor %}
-          </select>
-        
-        <div id="location" class="fake-border hidden">
-        <i class="glyphicon glyphicon-map-marker"></i>
-
+<h1 class="mt40 text-center meetup-list-head">Upcoming Meetups</h1>
+<p class="meetup-list-subhead">We host meetups, workshops, networking events, and more. See what’s happening near you.</p>
```

Comment:
Honestly you should let the spaces be and not let such changes be reflected.

Reply:
That complete part is being removed. That is why I removed the space after that as well

Reply:
Its okay :)